### ISSUE TITLE
Load Graph: 'Full screen graph view' control present under 'Graph' tab section is not accessible though keyboard.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1062]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1248955

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Gremlin(graph)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate  to "New Graph" menu item and select it.
Navigate through all the controls under Add Graph blade and enter details and select ok button to create new 'Graph'.
Create 'Sample Graph under 'Gremlin  API' section and expand created graph control. 
Navigate through all the controls under "Gremlin API" Section and select "Graph" which appears after expending the items.
Navigate through all the controls like JSON query edit field type query and select Load Graph.
Navigate to 'New Vertex' menu item and fill the details under 'New Vertex' blade and select ok button to get the data available under Graph and JSON tab section.
Navigate through all the controls under JSON and Graph Tab items using tab key.
Verify the issue.

Actual Result:
'Full screen graph view' control present under 'Graph' tab section is not accessible though keyboard due to this user won't be perform functionality over that control. 

Expected Result:
'Full screen graph view' control present under 'Graph' tab section which is accessible using mouse should also be accessible though keyboard so that the user will be able to perform functionality over such controls. 

Note:
'Tabstop' of 'Accessibility Insights for Web' is not working on this page.

**Comment by ato9000** [comment]
https://user-images.githubusercontent.com/8519407/132711756-1aeed431-0e65-4c26-bc54-b4f7358bfc84.mp4

**Comment by ato9000** [comment]
https://user-images.githubusercontent.com/8519407/132712014-02e44267-892d-4cd3-a56e-5bca4f61f8b3.mp4
